srcpkgs/vpn-slice/template

```diff
@@ -10,4 +10,6 @@ maintainer="Jony <maybe-one-day-ubermensch@protonmail.com>"
 license="GPL-3.0-or-later"
 homepage="https://github.com/dlenski/vpn-slice"
 distfiles="https://github.com/dlenski/${pkgname}/archive/v${version}.tar.gz"
-checksum=9e6f2fc9c1305d78e804529ddc7e787a88d185ba7a3560218ab5cdee4233a757
+checksum=ecdb1c434c611427447b4884ea46845584ab08e40687e3dd4ffbdb7cd9bbcbde
+# There aren't any tests defined
+make_check=no
```

Comment:
```suggestion
make_check=no # there aren't any tests defined
```

That way it's easier to see why tests are disabled when grepping for `make_check=`.

Reply:
Done!